common/types.go

```diff
@@ -0,0 +1,15 @@
+package common
+
+import (
+	"math/big"
+)
+
+type Hash [32]byte
+
+type BlockHeader struct {
```

Comment:
Please document each of these fields briefly

Reply:
done

Reply:
done

Reply:
I think it would be more conventional to comment them like this:
```
// BlockHeader is the header of a Polkadot block
type BlockHeader struct {
	ParentHash     Hash     // ParentHash is the block hash of the block's parent
	Number         *big.Int // Number is the block number
	StateRoot      Hash     // StateRoot is the root of the state trie
	ExtrinsicsRoot Hash     // ExtrinsicsRoot is the root of the extrinsics trie
	Digest         []byte   // any addition block info eg. logs
}
```

Reply:
ok I will change it